app/controllers/atomic_admin/atomic_tenant_client_id_strategy_controller.rb

```diff
@@ -1,14 +1,8 @@
 module AtomicAdmin
   class AtomicTenantClientIdStrategyController < ApplicationController
-    def pinned_client_id_params
-      params.permit(:iss, :client_id, :application_instance_id)
-    end
-
-    def find_pinned_client_id
-      AtomicTenant::PinnedClientId.find_by(id: params[:id])
-    end
+    include Filtering
 
-    def search 
+    def search
```

Comment:
maybe we could add a comment that this one is deprecated/only used by the old admin app. same with the other controllers

Reply:
Good call